Fix change user not working if a learner hadn't signed in yet

Burge337 opened the pull request:
Also includes minor improvements to options menu UI looks and behavior:
1) Close options menu button is now a big red X at the top right corner
2) Update Word Bank now also closes the menu

TODO: add "wordbank updated" message to the update word bank button so the user knows if it actually worked.

Burge337 committed:
Fix change user not working if a learner hadn't signed in yet

Also includes minor improvements to options menu UI looks and behavior:
1) Close options menu button is now a big red X at the top right corner
2) Update Word Bank now also closes the menu

TODO: add "wordbank updated" message to the update word bank button so the user knows if it actually worked.

Burge337 committed:
Add confirmation dialogue to the UpdateWordBank button

The text is placed over the top of the word bank for 2 seconds when clicking the button. I'm open to moving this text around; not convinced that over top of the word bank is the best place for it.

Burge337 committed:
Make options menu overlay selected scene

The options menu previously just opened in the sentence builder scene even if we'd pressed the change scene button.

Burge337 committed:
Make options menu more context sensitive

Now removes the updatewordbank button while in the storybuilder scene.

Burge337 committed:
Merge branch 'master' into options-menu-improvements

Burge337 committed:
Fix weird button positioning in learnerlogin scene

Copied over the master branch's setup of the change user and refresh learner icon buttons.

kklamberty reviewed (changes requested):
I would like to see the name of the learner who is logged in on both scenes. Maybe near the button that you use to switch views?

Burge337 committed:
Add display for current logged in learner

This adds the learner's profile picture and name to the bottom left of whatever scene the user is in. Slightly re positioned the open menu button so that it remains fully on screen when switch back and forth between scene. 

Also removes a silly optimization with changing around Time.fixedDeltaTime and moved the functions that grab learner sprites to the LearnerIconStorageHandler class.

Burge337 committed:
Merge branch 'master' into options-menu-improvements

Burge337 commented:
The options menu button is appearing over top of its own menu while in the sentence builder scene.
